lib/internal/errors.js

```diff
@@ -186,12 +187,26 @@ E('ERR_V8BREAKITERATOR', 'Full ICU data not installed. ' +
 function invalidArgType(name, expected, actual) {
   const assert = lazyAssert();
   assert(name, 'name is required');
-  const type = name.includes('.') ? 'property' : 'argument';
-  var msg = `The "${name}" ${type} must be ${oneOf(expected, 'type')}`;
-  if (arguments.length >= 3) {
-    msg += `. Received type ${actual !== null ? typeof actual : 'null'}`;
+  assert.strictEqual(arguments.length, 3, 'arguments length mismatch');
+  const argType = name.includes('.') ? 'property' : 'argument';
+  // Only check the first entry
+  const expectedType = Array.isArray(expected) ? expected[0] : expected;
+  // A-Z = 65-90, a-z = 97-122
+  const type = expectedType.charCodeAt(0) > 96 ? 'of type' : 'instance of';
```

Comment:
An alternative way to do this would be to pass a class instead of a string. This might be more obvious?